modules/appmodel/app-transform-generator.c

```diff
@@ -133,7 +133,7 @@ app_transform_generator_new(gint context, const gchar *name)
   AppTransformGenerator *self = g_new0(AppTransformGenerator, 1);
 
   app_object_generator_init_instance(&self->super, context, name);
-  self->super.parse_arguments = app_transform_generator_parse_arguments;
-  self->super.generate_config = app_transform_generate_config;
+  self->super.parse_arguments = _parse_arguments;
+  self->super.generate_config = _generate_config;
```

Comment:
Optional: I think if one chooses to use long private method names, we should leave them as is.
The short names in our codebase have already started to collide with each other, which makes debugging a bit less pleasant.

Reply:
I'm okay with either. We can keep already named functions. I had to add a free method, and it seemed strange to see long and short function names close to each other.

Reply:
Dropped the commit.

Reply:
I've decided that I personally will use long names when the chance of collision is higher (for example, `_init()`).

Mixing short and long names may look strange, but those common short names cause some minor headache in gdb.